### PR TITLE
Fix warnings due to the division operator being deprecated

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: scripts/prepare-package.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ function validateSkyuxIconVersionMatch() {
 function renderScss(file, target) {
   const result = sass.renderSync({
     file: file,
-    importer: tildeImporter
+    importer: tildeImporter,
+    quietDeps: true
```
Comment (Blackbaud-TrevorBurch) [inline scripts/prepare-package.js:58]: Along with updating our variables I am enabling this flag in order to suppress deprecation warnings that are coming from our upstream dependencies. See https://sass-lang.com/documentation/cli/dart-sass#quiet-deps for more information on this flag. 

Without this flag - we get a warning from our import of `~normalize-scss/sass/normalize/import-now`

Comment (Blackbaud-PaulCrowder) [reply scripts/prepare-package.js:58]: Will we still know if we're using something that's deprecated?

Comment (Blackbaud-TrevorBurch) [reply scripts/prepare-package.js:58]: It does not appear that this dependency is being actively updated and so we don't have a way to fix this warning without this flag unless we research removing this import and finding an alternative.

Comment (Blackbaud-PaulCrowder) [reply scripts/prepare-package.js:58]: Yeah, I'm fine with suppressing messages from things we depend on, but I want to make sure that if _we_ start using something that's deprecated in our own code that this doesn't suppress those warnings.

Comment (Blackbaud-TrevorBurch) [reply scripts/prepare-package.js:58]: Correct. I double checked that if we have that flag on and revert my other changes that the warnings are still output.

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Pull request merged**